.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11fortran_compiler_version11mpi_typeconda.yaml

```diff
@@ -1,29 +1,27 @@
 c_compiler:
 - gcc
 c_compiler_version:
-- '12'
+- '11'
```

Comment:
This was using GCC 12 before. Am guessing that is just because [12 is the default GCC version used]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/945b55c62ecb039ff67c1860fb65ed5b05937f08/recipe/conda_build_config.yaml#L6-L7 ) if not otherwise overridden

Whereas CUDA 11.8, is configured with [GCC 11]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/945b55c62ecb039ff67c1860fb65ed5b05937f08/recipe/migrations/cuda118.yaml#L80-L81 )

Not sure whether this matters for OpenMPI in practice. So just mentioning for clarity